pages/api/auth/[...nextauth].ts

```diff
@@ -97,21 +96,7 @@ export const authOptions: NextAuthOptions = {
       session.user.acceptableUse = token.acceptableUse;
       session.user.name = token.name ?? null;
       session.user.image = token.picture ?? null;
-
-      const privileges = await getPrivilegeRulesForUser(token.userId as string);
-      if (!privileges.length) throw new Error("User has no privileges");
```

Comment:
@bryan-robitaille removed it from here since the `getPrivilegeRulesForUser` already does that check (and also throws an error)